### ISSUE TITLE
Site in inaccessible 

### ISSUE THREAD

**Issue by Zemerik** [JadeBetty/jadebetty.github.io#1]
Hey, currently I'm unable to view your website. 

![image](https://github.com/JadeBetty/jadebetty.github.io/assets/133865660/35f7892a-c9c1-45e5-9f4e-34ea4f534c3a)

**Comment by JadeBetty** [comment]
yeah im really sorry for that, this site has been archived, idk what to do with this anymore
